app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,284 @@
+                android:fontFamily="@font/robotolight"
+                android:layout_marginTop="4dp"
+                android:paddingTop="6dp"
+                android:paddingBottom="6dp"
+                app:layout_constraintStart_toStartOf="@id/storyline"
+                app:layout_constraintTop_toBottomOf="@id/storyline"
+                app:layout_constraintEnd_toEndOf="parent"
+                android:layout_marginEnd="16dp"/>
+            <TextView
+                android:id="@+id/cast_Text_View"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                app:layout_constraintHorizontal_weight="1"
+                android:text="@string/cast"
+                android:textColor="@color/white"
+                android:textStyle="bold"
+                android:textSize="14dp"
+                android:fontFamily="@font/robotoblack"
+                android:paddingBottom="7dp"
+                android:layout_marginTop="24dp"
+                app:layout_constraintEnd_toEndOf="@id/description"
+                app:layout_constraintStart_toStartOf="@id/description"
+                app:layout_constraintTop_toBottomOf="@id/description"/>
+
+            <ImageView
+                android:id="@+id/first_actor_image_view"
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                android:src="@drawable/img_one"
+                app:layout_constraintDimensionRatio="1:1"
+                app:layout_constraintHorizontal_weight="1"
+                app:layout_constraintHorizontal_chainStyle="spread_inside"
+                app:layout_constraintStart_toStartOf="@id/cast_Text_View"
+                app:layout_constraintEnd_toStartOf="@id/second_image_view"
+                app:layout_constraintTop_toBottomOf="@id/cast_Text_View" />
+
+            <ImageView
+                android:id="@+id/second_image_view"
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                android:src="@drawable/img_two"
+                app:layout_constraintDimensionRatio="1:1"
+                app:layout_constraintHorizontal_weight="1"
+                android:layout_marginStart="8dp"
+                android:layout_marginEnd="8dp"
+                app:layout_constraintHorizontal_chainStyle="spread_inside"
+                app:layout_constraintStart_toEndOf="@id/first_actor_image_view"
+                app:layout_constraintEnd_toStartOf="@id/third_image_view"
+                app:layout_constraintTop_toBottomOf="@id/cast_Text_View" />
+            <ImageView
+                android:id="@+id/third_image_view"
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                android:src="@drawable/img_three"
+                app:layout_constraintHorizontal_weight="1"
+                app:layout_constraintDimensionRatio="1:1"
+                android:layout_marginEnd="8dp"
```

Comment:
remove
android:layout_marginEnd="8dp"
and add
android:layout_marginStart="8dp" to fourth_image_view